src/services/blt_rewards.py

```
@@ -0,0 +1,793 @@
+    else:
+        if author_index > 0:
+            medal = ["🥇", "🥈", "🥉"][author_index - 1] if author_index - 1 < 3 else ""
+            comment += row_for(author_index, sorted_users[author_index - 1], False, medal) + "\n"
+        medal = ["🥇", "🥈", "🥉"][author_index] if author_index < 3 else ""
+        comment += row_for(author_index + 1, sorted_users[author_index], True, medal) + "\n"
+        if author_index < len(sorted_users) - 1:
+            comment += row_for(author_index + 2, sorted_users[author_index + 1]) + "\n"
+    comment += "\n---\n"
+    comment += (
+        f"**Scoring this month** (across {owner} org): Open PRs (+1 each), Merged PRs (+10), "
+        "Closed (not merged) (−2), Reviews (+5; first two per PR in-month), "
```

Ruff (0.15.7)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)